### ISSUE TITLE
forgot to use std_ thingy in utils::impossible

### ISSUE THREAD

**Issue by Monadic-Cat** [rodrimati1992/core_extensions#10]
In `src_core_extensions/util.rs`, there is this on line 177:
```rust
std::hint::unreachable_unchecked()
```
and it should be
```rust
std_::hint::unreachable_unchecked()
```

This broke my build when I went to update all my dependencies, lol.
(this change would be a PR if it were anything less trivial, but it's literally a single typo fix)

**Comment by rodrimati1992** [comment]
Thanks for pointing this out, I'll fix this within 24 hours and add testing of `#[cfg(not(debug_assertions))]` code like what broke your build.

**Comment by rodrimati1992** [comment]
Fixed this in 1.3.3
